testomania/src/main/java/com/earth/testomania/opentdb/OpenTdbRepo.kt

```diff
@@ -0,0 +1,44 @@
+package com.earth.testomania.opentdb
+
+import com.earth.testomania.opentdb.models.OpenTdbQuizDto
+import retrofit2.Response
+import javax.inject.Inject
+
+class OpenTdbRepo @Inject constructor(private val api: OpenTdbQuizApi) {
```

Comment:
I would suggest creating interface for `OpenTdbRepository` and we can rename this class to  `OpenTdbRepositoryImpl` which will comply with our other code naming convention